apps/prairielearn/src/lib/groups.sql

```diff
@@ -77,13 +77,13 @@ WITH
       gur.group_role_id
   )
 SELECT
-  gr.id,
-  gr.role_name,
-  COALESCE(gur.count, 0) AS count,
-  gr.maximum,
-  gr.minimum,
-  gr.can_assign_roles_at_start,
-  gr.can_assign_roles_during_assessment
+  gr.*,
+  COALESCE(gur.count, 0)::INT AS count,
+  -- The can_assign_roles column is not yet guaranteed to be up-to-date, as the
+  -- course may not have synced with the new code yet. Use the old
+  -- can_assign_roles_at_start column instead for now. This will be replaced in
+  -- a future version.
+  gr.can_assign_roles_at_start AS can_assign_roles
```

Comment:
Is it on your radar to fast-follow and fix this? If there's a chance you might not immediately get to this, could you file an issue to track this cleanup work?